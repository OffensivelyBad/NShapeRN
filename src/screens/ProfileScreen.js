import React, { PureComponent } from 'react';
import { View, Text } from 'react-native';

class ProfileScreen extends PureComponent {
  render() {
    return (
      <View><Text>This is the profile screen This is the profile screen This is the profile screen This is the profile screen</Text></View>
    );
  }
}

export default ProfileScreen;
