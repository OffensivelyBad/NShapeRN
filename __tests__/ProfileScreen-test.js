import React from 'react';
import ProfileScreen from '../src/screens/ProfileScreen';
import renderer from 'react-test-renderer';

test('ProfileScreen renders correctly', () => {
    const tree = renderer.create(<ProfileScreen />).toJSON();
    expect(tree).toMatchSnapshot();
});
