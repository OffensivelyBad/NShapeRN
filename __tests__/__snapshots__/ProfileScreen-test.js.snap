// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen renders correctly 1`] = `
<View>
  <Text>
    This is the profile screen This is the profile screen This is the profile screen This is the profile screen
  </Text>
</View>
`;
